lightwood/api/types.py

```diff
@@ -561,6 +561,7 @@ class PredictionArguments:
     fixed_confidence: Union[int, float, None] = None
     anomaly_error_rate: Union[float, None] = None
     anomaly_cooldown: int = 1
+    forecast_offset: int = 0
```

Comment:
did this automatically default to 0 before?

Reply:
It did not exist before, this is new because sktime explicitly needs either an absolute or relative reference to the latest data point seen in training. This PR proposes using a relative reference.